### DISCUSSION TITLE
Underwater fearures

### DISCUSSION THREAD

**Discussion by 4wdiver** [gopro/labs#327] [Feature Requests]
Hello. I have a project. Making my movie documentary of a solo scubadiver girl traveling on a 4wd to dive to the most inhospitable sites in the world. I have all my gear to film on land and air. I have a GoPro 9 and I was tempted to buy an 11 since the capacity for shooting at 10bits . However I feel a lack of some features to be perfect for underwater filmmakers and have gopro as a standard camera allowed on Netflix. I’m hopping for a rear screen with some capabilities to be access trough the underwater case. Or instead a button for white balancing calibration. That simple issue with the flat or log film capabilities and 4K 10bits Gopro Will be unbeatable. What you guys think or a filmmaker think of another lack on gopro to improve underwater

**Comment by dnewman-gpsw** [comment]
Labs feature do solve some underwater shooting issues, via using laminated QR Codes to access any setting during a dive.  The one feature I think you need to white balance off a card.  This is not available, but is a good feature to request.  What else is needed?   HERO11 with 10-bit, and Labs added custom log curve and a wide color gamut, would likely be ideal.

**Reply by 4wdiver** [reply]
Thank you. Yes it will be ideal for underwater filmmakers. A custom log curve it’s great to retrieve some information on lowlight and get it after editing colorgrading. Raw wil reduções noise.  But for sure a qr card to balance white balance underwater that would be huge. Thank you
